src/ProjectTemplates/test/ProjectExtensions.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Templates.Test.Helpers;
+
+namespace Templates.Test;
+
+public static class ProjectExtensions
+{
+    public static async Task<Project> VerifyLaunchSettings(this Project project, string[] expectedLaunchProfileNames)
+    {
+        var launchSettingsFiles = Directory.EnumerateFiles(project.TemplateOutputDir, "launchSettings.json", SearchOption.AllDirectories);
```

Comment:
Should this also be checking that there aren't extra launch profiles in here, as I saw some comments in some tests saying stuff like https should not be in the launch settings?

Reply:
Yep good point, I'll change it verify the expected profiles exactly including order.